src/pages/schedule-interview/components/CandidateConfirmation/__test__/CandidateConfirmation.test.tsx

```diff
@@ -75,11 +87,31 @@ describe("CandidateConfirmation", () => {
     expect(screen.getByText("https://example.com/interview")).toBeInTheDocument()
   })
 
-  it('has an "Add to calendar" button', () => {
+  it("should render correct buttons", () => {
+    render(<CandidateConfirmation />)
+    const addToCalendarButton = screen.getByRole("button", { name: /add to calendar/i })
+    const cancelInterviewButton = screen.getByRole("button", { name: /cancel interview/i })
+    const rescheduleInterviewButton = screen.getByRole("button", { name: /reschedule interview/i })
+
+    expect(addToCalendarButton).toBeInTheDocument()
+    expect(cancelInterviewButton).toBeInTheDocument()
+    expect(rescheduleInterviewButton).toBeInTheDocument()
+  })
+
+  it("should open cancel interview modal when cancel interview button is clicked", async () => {
+    const user = userEvent.setup()
+    const { openCancelInterviewModal } = mockUseCandidateCancelInterview()
+
     render(<CandidateConfirmation />)
-    const button = screen.getByRole("button", { name: /add to calendar/i })
-    expect(button).toBeInTheDocument()
-    expect(button).toHaveClass("bg-black")
+    const cancelInterviewButton = screen.getByRole("button", { name: /cancel interview/i })
+
+    act(() => {
+      void user.click(cancelInterviewButton)
+    })
```

Comment:
I think the recommended way is to `await` it?

```suggestion
      await user.click(cancelInterviewButton)
```